app/models/deed.rb

```diff
@@ -59,4 +59,11 @@ def update_collections_most_recent_deed
       self.collection.update(most_recent_deed_created_at: self.created_at)
     end
   end
+
+  def update_works_most_recent_deed
```

Comment:
This looks like the correct logic, but there is no callback that actually fires this.  (See line 20 for an example.)